index.js

```diff
@@ -1,4 +1,4 @@
-const SHORTSPLIT = /$|[!-@\[-`{-~]/
+const SHORTSPLIT = /[!-@\[-`{-~].*|$/g
```

Comment:
@jridgewell What's the idea behind moving `|$` to the end and adding `g`? We only care about the first match.

Reply:
@jridgewell `lastIndex`, got it! 👍 You demon! 😈

Reply:
> What's the idea behind moving `|$` to the end

My first commit removed it, then I added it back 😅, but put it at the end (where I usually would put it). I'll move that back.

> adding `g`

There can only ever be one match, but the `g` flag allows `lastIndex` to work. Without it, `RegExp#exec` starts from the beginning every time.